README.md

```diff
@@ -1,4 +1,7 @@
 # Roll For Reaction
```

Comment:
If we have the logo (which contains the title of the project), I don't think this heading is necessary anymore.